Sources/ProjectDescription/WorkspaceGenerationOptions.swift

```diff
@@ -31,15 +31,20 @@ extension Workspace {
         /// It is recommended to set the version option to Xcode's version that is used for development of a project, for example `.lastXcodeUpgradeCheck(Version(13, 0, 0))` for Xcode 13.0.0.
         public let lastXcodeUpgradeCheck: Version?
 
+        /// Allows to render markdown files inside the workspace including an .xcodesamples.plist inside it.
+        public let renderMarkdownReadme: Bool
```

Comment:
Is it specific to README or it is for markdown in general? If generic I would rename it to `renderMarkdown`

Reply:
It's only for the README.md or readme files :/